sbt-sample/build.sbt

```diff
@@ -8,16 +8,22 @@ crossScalaVersions := Seq("2.12.0-RC1", "2.11.8", "2.10.6")
 
 scalaVersion := "2.12.0-RC1"
 
-// add scala-xml dependency when needed (for Scala 2.11 and newer) in a robust way
-// this mechanism supports cross-version publishing
+// add dependencies on standard Scala modules, in a way
+// supporting cross-version publishing
+// taken from: http://github.com/scala/scala-module-dependency-sample
 libraryDependencies := {
   CrossVersion.partialVersion(scalaVersion.value) match {
-    // if scala 2.11+ is used, add dependency on scala-xml module
-    case Some((2, scalaMajor)) if scalaMajor >= 11 =>
+    // if Scala 2.12+ is used, use scala-swing 2.x
+    case Some((2, scalaMajor)) if scalaMajor >= 12 =>
       libraryDependencies.value ++ Seq(
         "org.scala-lang.modules" %% "scala-xml" % "1.0.6",
         "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.4",
         "org.scala-lang.modules" %% "scala-swing" % "2.0.0-M2")
+    case Some((2, scalaMajor)) if scalaMajor >= 11 =>
+      libraryDependencies.value ++ Seq(
+        "org.scala-lang.modules" %% "scala-xml" % "1.0.6",
+        "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.4",
+        "org.scala-lang.modules" %% "scala-swing" % "1.0.2")
     case _ =>
```

Comment:
Yes, the snippet from the build.sbt should match the README.  My thoughts exactly.  Thanks for doing this.